### ISSUE TITLE
Barrierefreiheit Stepper (OZG): Transparenz ändern

### ISSUE THREAD

**Issue by SimoneReuber** [IHK-GfI/lux-components-theme#71]
Beschreibung:

Das axe Accessibility Testing Tool hat folgende Aspekte der Applikation als benutzerunfreundlich gemeldet:

Elements must have sufficient color contrast

"Element has insufficient color contrast of 3.86 (foreground color: #6685a3, background color: #ffffff, font size: 16.5pt (22px), font weight: normal). Expected contrast ratio of 4.5:1"
"Element has insufficient color contrast of 3.7 (foreground color: #6080a0, background color: #eff3f6, font size: 16.5pt (22px), font weight: normal). Expected contrast ratio of 4.5:1"

Die ausgegrauten Steps im Stepper haben keinen ausreichenden Kontrast (Sowohl die Labels als auch die Nummerierungen).

![image](https://github.com/IHK-GfI/lux-components-theme/assets/168449931/9a9eb133-08e7-441b-bd00-d4a73d503ad6)

Vorschlag:

Aktuell werden noch nicht ausgefüllte Steps ausgegraut, indem deren Transparenz auf 60% gesetzt wird. Ohne diese Transparenz wären diese Steps besser zu erkennen (siehe Screenshot).

![image](https://github.com/IHK-GfI/lux-components-theme/assets/168449931/f5f944be-37c4-46a2-b8e9-15f20d6cd7d8)

Certain ARIA roles must contain particular children
"Element has children which are not allowed (see related nodes) Element has no aria-busy="true" attribute"

Der Applikations-Stepper besitzt das Attribut role="tablist", weshalb dieser ausschließliche Unterelemente mit dem Attribut role="tab" besitzen sollte.

Da dieses Element in den Stepper integriert ist, kann eine Anpassung nur in den Lux-Components erfolgen.

 Abgrenzung 

Es gibt weitere Anpassungen im Farbe/Kontrast --> WLA-721 (OZG-Jira-Board)

ALLE DATEIEN:

[[#WLA-721] Kontraste von Texten ausreichen.pdf](https://github.com/IHK-GfI/lux-components-theme/files/15150599/WLA-721.Kontraste.von.Texten.ausreichen.pdf)

[[#WLA-720] CLONE - Fix Barrierefreiheit Stepper.pdf](https://github.com/IHK-GfI/lux-components-theme/files/15150601/WLA-720.CLONE.-.Fix.Barrierefreiheit.Stepper.pdf)

**Comment by SimoneReuber** [comment]
[[#WLA-720] CLONE - Fix Barrierefreiheit Stepper.pdf](https://github.com/IHK-GfI/lux-components-theme/files/15150696/WLA-720.CLONE.-.Fix.Barrierefreiheit.Stepper.pdf)

**Comment by patrowe** [comment]
Die Anpassungen für den Farbkontrast, werden gemäß des Vorschlags umgesetzt.

Für die Anpassungen der ARIA-Attribute bitten wir um weitere Angaben, den Fehler konnten wir mit dem Tool AXE in unserer Demo-Applikation nicht nachstellen.

**Comment by thodic77** [comment]
> Die Anpassungen für den Farbkontrast, werden gemäß des Vorschlags umgesetzt.

Die Anpassungen wurden umgesetzt.

> Für die Anpassungen der ARIA-Attribute bitten wir um weitere Angaben, den Fehler konnten wir mit dem Tool AXE in unserer Demo-Applikation nicht nachstellen.

Meeting 07.05.2024: Das ARIA-Attribute-Problem kann in der aktuellen Version nicht nachgestellt werden und basiert auf einem über einen Jahr alten Test. Aus diesem Grund wurde beschlossen, diese Anforderung im Rahmen dieses Issues nicht weiter zu verfolgen. 

@patrowe: Bitte testen.

**Comment by patrowe** [comment]
Test durchgeführt unter Windows 10 in den Browsern
Microsoft Edge: -> OK
Firefox: -> OK
Chrome: -> OK

Test durchgeführt unter MacOs Sonoma 14.5 in den Browsern
Safari: -> OK
Firefox: -> OK
Chrome: -> OK

Die Anforderungen wurden gemäß der Akzeptanzkriterien umgesetzt.
